gulp-tasks/utils/constants.js

```diff
@@ -18,4 +15,12 @@ module.exports = {
   // TODO: Change to v3.0.0 when v3 is launched to avoid doing anything
   // with prelease versions.
   MIN_RELEASE_TAG_TO_PUBLISH: 'v3.0.0-alpha',
+  GITHUB_OWNER: 'GoogleChrome',
+  GITHUB_REPO: 'workbox',
+
+  // This is the namespace of the workbox module.
+  NAMESPACE_PREFIX: 'google.workbox',
```

Comment:
Only reason I did this (And I'm not too fussed either way) is that I had imaginged the common use case with the `workbox-loader` / CDN will be that we'd ship a 'Workbox' class that you'd use as:

```javascript
const workbox = new Workbox();
workbox.precaching.precache([]);
workbox.router.registerRoute(.....);
```

With this in mind, it gets messy if we use `workbox.` as the name space.

Happy to change it though.